<launch>
    <node name="startlight_watcher" pkg="rr_iarrc" type="startlight_watcher" output="screen" required="true">
        <!--Input and output topics-->
        <param name="img_topic" type="string" value="/camera_center/image_color_rect"/>
        <param name="stoplight_watcher_topic" type="string" value="/start_detected"/>

        <!--Controls the sensitivity of circle detection (lower = more circles detected)-->
        <param name="circularity_threshold" type="double" value="0.7"/>
    </node>
</launch>
